hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -0,0 +1,155 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "entropy_src",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  bus_host: "none",
+  interrupt_list: [
+    { name: "es_noise_valid"
+      desc: "Asserted when noise source bits are available."}
+    { name: "es_noise_fifo_err"
+      desc: "Asserted when a FIFO error occurs."}
+  ]
+  regwidth: "32",
+  registers: [
+    { name: "ESCONF",
+      desc: "ENTROPY_SRC configuration register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "0",
+          name: "Master enable",
```

Comment:
These field names become SW header entries, so you probably want a) short names b) no spaces. So perhaps `EN` or `ENABLE` here.

Reply:
Noticed this earlier as being a problem. Fixed.